UpdateDependencies.ps1

```diff
@@ -0,0 +1,129 @@
+﻿#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+function EnsureEnvironmentVariable([string]$envVarName)
+{
+    If ([Environment]::GetEnvironmentVariable($envVarName) -eq $null)
+    {
+        throw "Can't find environment variable '$envVarName'"
+    }
+}
+
+function SetEnvIfDefault([string]$envVarName, [string]$value)
+{
+    If ([Environment]::GetEnvironmentVariable($envVarName) -eq $null)
+    {
+        [Environment]::SetEnvironmentVariable($envVarName, $value)
+    }
+}
+
+EnsureEnvironmentVariable "GITHUB_USER"
```

Comment:
The reason to use environment variables is because that is the easiest way to pass these variables in using a VSO build. The main runner of this script will be a VSO build definition, where all you need to do is define variables with the same names and these environment variables are set to the variables defined in VSO.

We do it this way for multiple CLI scripts that are run by VSO.